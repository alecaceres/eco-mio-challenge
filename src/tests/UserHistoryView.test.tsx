import { ThemeProvider } from '@mui/material/styles';
import { lightTheme } from '@src/theme';
import { UserHistoryView } from '@src/views/user/UserHistoryView';
import renderer from 'react-test-renderer';

it('if dom manipulation view snapshot matches', () => {
  const tree = renderer
    .create(
      <ThemeProvider theme={lightTheme}>
        <UserHistoryView />
      </ThemeProvider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
