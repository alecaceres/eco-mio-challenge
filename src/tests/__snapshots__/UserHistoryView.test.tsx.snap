// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`if dom manipulation view snapshot matches 1`] = `
<div>
  I could be a table
</div>
`;
